AEPCore/Sources/core/MobileCore.swift

```diff
@@ -38,6 +39,40 @@ public final class MobileCore: NSObject {
     /// Pending extensions to be registered for legacy support
     static var pendingExtensions = ThreadSafeArray<Extension.Type>(identifier: "com.adobe.pendingExtensions.queue")
 
+    public static func start(with appID: String,
+                             logLevel: LogLevel? = nil,
+                             configUpdates: [String: Any]? = nil,
+                             additionalContextData: [String: Any]? = nil,
+                             disableLifecycleStart: Bool? = nil,
+                             applicationState: UIApplication.State? = nil
```

Comment:
I think we can detect this ourself instead of requiring it as a param.

Reply:
UIApplication.shared instance is not available in Application Extensions. Let me look into this though.